Item23_Understand_std_move_and_std_forward/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-set_source_files_properties(std_move_in_cpp14.cpp PROPERTIES COMPILE_FLAGS "-std=c++14 -stdlib=libc++")
+set_source_files_properties(std_move_in_cpp14.cpp PROPERTIES COMPILE_FLAGS "-std=c++14")
```

Comment:
Yo Bart ! I've created a new branch 'clang' directly clone from 'gcc-4.9' with some scipts (configure/build) modifications. 
=> https://github.com/yoyonel/Effective-Modern-Cpp/tree/clang-3.5
It seems to build correctly (with no more efforts). 
With my clang-3.5 on my system, the flag '-stdlib=libc++' seems not to be necessary.